pkg/templates/charts/toggle/console/templates/acm-cli-consoleclidownload.yaml

```
@@ -12,15 +12,18 @@ metadata:
   name: acm-cli-downloads
 spec:
   description: |
-    With the CLIs for Red Hat Advanced Cluster Management for Kubernetes (RHACM) you 
+    With the CLIs for Red Hat Advanced Cluster Management for Kubernetes you 
     can enhance your multicluster experience at the terminal. This includes:
     - `policytools`
-        - Interact with RHACM policies locally, including resolving templates locally.
+        - Interact with Red Hat Advanced Cluster Management policies locally, 
+          including resolving templates locally and (Dev Preview) executing 
+          a ConfigurationPolicy locally.
```

Does ConfigurationPolicy need to be together? Also when you say execute, do you mean "running" a policy or "applying" a policy? 
```suggestion
        - Interact with Red Hat Advanced Cluster Management policies locally, 
          such as resolving templates locally and running 
          a configuration policy locally (Dev Preview).
```

---

It does a dry run of what the controller would do. Inside of the controller, it's called "reconciling", so maybe we use that terminology here also?